pyispyb/core/routes/userportalsync.py

```diff
@@ -0,0 +1,47 @@
+import logging
```

Comment:
As i suggested to @mgaonach for the SSX population routes. I would suggest moving these routes into `/routes/webservices`  so it is clear that these routes are to be called by other services rather than the UI.
I also think its sensible to add a db option to disable these, i personally don't want these routes sitting exposed to the internet!

Reply:
Added by https://github.com/ispyb/py-ispyb/pull/210/commits/955b9f14a8a85f39303530f97b4da6757bb30164